bindings/pyroot/pythonizations/test/pythonizedDefineAndFilters/df001_introduction_new.py

```diff
@@ -0,0 +1,58 @@
+import ROOT
```

Comment:
I would not create a whole new tutorial, but rather introduce the functionality in some RDF operation here. For example, in this tutorial you have several filters, so you can show how filter can accept a string with C++ code and also a Python function/lambda -- and this should be explained with comments in the tutorial.